src/salte-auth.js

```diff
@@ -44,6 +44,7 @@ const logger = debug('@salte-io/salte-auth');
  * @property {('session'|'local')} [storageType='session'] The Storage api to keep authenticate information stored in.
  * @property {Boolean|Validation} [validation] Used to disable certain security validations if your provider doesn't support them.
  * @property {Boolean} [autoRefresh=true] Automatically refreshes the users token upon switching tabs or one minute prior to expiration.
+ * @property {Number} autoRefreshBuffer A number of miliseconds before token expiration to refresh.
```

Comment:
Only issue I can see is that `autoRefreshBuffer` should be `[autoRefreshBuffer=60000]`.

This just denotes to the user that `autoRefreshBuffer` is optional and that it defaults to 60000.